rllib/policy/eager_tf_policy_v2.py

```diff
@@ -470,9 +474,11 @@ def compute_actions_from_input_dict(
         self._is_recurrent = state_batches != []
 
         # Call the exploration before_compute_actions hook.
-        self.exploration.before_compute_actions(
-            timestep=timestep, explore=explore, tf_sess=self.get_session()
-        )
+        if getattr(self, "exploration", None):
```

Comment:
is `if self.exploration:` something you can do instead of `if getattr(self, "exploration", None)`? doesn't `self.exploration` always exist? you set it either to None or something else in your constructor.